developer/frameworks/articles/09-info-framework/01-the-info-framework-intro.markdown

```diff
@@ -0,0 +1,81 @@
+# The Info Framework
+
+The Info Framework provides enhances flexibility for retrieving, processing, 
+and displaying any information, regardless of where the information comes from. 
+The Info Framework can process data from the database, created 
+through process in memory, or retrieved from an external source.
+
+The Info Framework is a new feature @product-ver@ and currently has limited 
```

Comment:
The phrase "limited functionality" was pulled directly from your draft: "While this is still a first version, with limited functionality..." Is there a phrase that you would rather use there?

Reply:
Saying that it has "limited functionality" is fine, since it's the truth. What I was saying is that we should not start by just saying that. We need to state what the Info Framework is first.